pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -267,6 +269,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 			// Just like the behaviour in case of static policy, takeByTopology will try to first allocate CPUs from the same socket
 			// and only in case the request cannot be sattisfied on a single socket, CPU allocation is done for a workload to occupy all
 			// CPUs on a physical core. Allocation of individual threads would never have to occur.
+			metrics.CPUManagerPinningErrorsTotal.Inc()
```

Comment:
Can we just use a defer() in this function instead of calling these explicitly at each return.

I.e. something like:
```
func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Container) (rerr error) {
    defer func() {
        if rerr == nil {
            metrics.CPUManagerPinningRequestsTotal.Inc()
        } else {
            metrics.CPUManagerPinningErrorsTotal.Inc()
        }
    }
```

Reply:
sure thing, trying this out this approach